packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -39,25 +58,42 @@ export async function createServer(
   const schema = await TypeGraphQL.buildSchema({
     container: Container,
     resolvers: [AccountResolver],
+    validate: false,
   });
+  const authHeader = config.authHeader.toLowerCase();
   const authUser = Container.get(SessionTokenAuth);
   const debugMode = config.env !== 'production';
-  const defaultContext = async ({ req }: any) => {
-    const bearerToken = req.headers[config.authHeader.toLowerCase()];
+  const defaultContext = async ({ req }: { req: Request }) => {
+    const bearerToken = req.headers[authHeader];
+    if (typeof bearerToken !== 'string') {
+      throw new AuthenticationError(
+        'Invalid authentcation header found at: ' + authHeader
+      );
+    }
     const userId = await authUser.lookupUserId(bearerToken);
     return {
       authUser: userId,
       token: bearerToken,
       logger,
     };
+    '';
   };
 
   return new ApolloServer({
     context: context ?? defaultContext,
     dataSources: () => ({
       authAPI: new AuthServerSource(),
+      profileAPI: new ProfileServerSource(),
     }),
-    formatError: err => reportGraphQLError(debugMode, logger, err),
+    formatError: (err) => formatError(debugMode, logger, err),
     schema,
+    uploads: false,
```

Comment:
Disables apollo's upload property which isn't needed when using the `graphql-upload` middleware package with express.